### ISSUE TITLE
storage slots can't be fetched on a fresh EVM

### ISSUE THREAD

**Issue by jalbrekt85** [paradigmxyz/pyrevm#14]
v.0.3.0

im trying to access the storage slots of an account via `EVM.storage()`, but it seems it'll just return `None` for all slots until the specific slot that is trying to be fetched is touched via `message_call`. is this intended? ideally we would want to be able to query the slot value from a fresh `EVM` instance

using weth as an example and trying to fetch [slot 0](https://evm.storage/eth/19665331/0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2/0x0000000000000000000000000000000000000000000000000000000000000000):

```py
from pyrevm import EVM

caller = "0xd8dA6BF26964aF9D7eEd9e03E53415D37aA96045"
weth = "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2"
calldata = bytes.fromhex("06fdde03") # calling "name()"

evm = EVM(fork_url="", fork_block="latest")

print(evm.storage(weth, 0)) # prints None

evm.message_call(caller=caller, to=weth, calldata=calldata) # touches slot 0

print(evm.storage(weth, 0)) # prints the slot value now
```

**Comment by DaniPopes** [comment]
All slots are `None` by default, and when touched (any read/write) they default to 0

I think this is fair behavior, you can treat both as the same, or we could just drop the option return and `unwrap_or_default` here

https://github.com/paradigmxyz/pyrevm/blob/99acd2ad4c5b455bcce2943065073723191cf90f/src/evm.rs#L125-L129

cc @DanielSchiavini

**Comment by DanielSchiavini** [comment]
@DaniPopes that doesn't make sense when running in a fork. I assumed the `load_account` function call would load the existing state from the `EthersDB`.

**Comment by jalbrekt85** [comment]
> when touched (any read/write) they default to 0
> you can treat both as the same

not sure what you mean by this? 

Intuitively, I would assume it would grab the on-chain state when forking. if this is the intended behavior, if should be more clear

**Comment by DaniPopes** [comment]
I didn't see the fork, should be fixed with #19.
